.github/workflows/publish.yml

```diff
@@ -0,0 +1,68 @@
+name: Wheel building
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+
+  test_wheel_building:
+    # This ensures that a couple of target wheels build fine in pull
+    # requests and pushes
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
+    if: (github.event_name == 'push' || github.event_name == 'pull_request')
```

Comment:
You want to do this for all the PRs? No special label?

Reply:
Yes, but it only does 2 wheels.  I copied this from astropy.  Perhaps it's unnecessary.